### DISCUSSION TITLE
验证码测试经常过不出来怎么办

### DISCUSSION THREAD

**Discussion by imRoxyy** [mikumifa/biliTickerBuy#246] [提问专区]
如图
<img width="1078" alt="3" src="https://github.com/mikumifa/biliTickerBuy/assets/174174263/e2725a4e-2203-483c-86a0-477994ead76f">

**Comment by mikumifa** [comment]
给报错

**Reply by imRoxyy** [reply]
没报错 就是有点抽风 现在又能出来了 刚刚刷新好几遍都出不来

**Comment by mikumifa** [comment]
不用担心这个，真真抢票时候会没问题的
